src/shared/utils/analytics.ts

```diff
@@ -35,9 +35,9 @@ export const analytics = configureAnalyticsClient<AnalyticsBrowser>({
 export function decorateAnalyticsEventsWithContext(
   getEventContextProperties: () => Record<string, unknown>
 ) {
-  void analytics.client.ready(
+  void segmentClient.ready(
```

Comment:
Can you explain this change?

Reply:
I'm hitting some odd type inference issues - for some reason `analytics.client` is not correctly typed, but since they share `segmentClient` then I'm just using that instead.